src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Repository/DataSetVersionImportRepository.cs

```diff
@@ -0,0 +1,31 @@
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Repository.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Repository;
+
+public class DataSetVersionImportRepository(PublicDataDbContext publicDataDbContext) : IDataSetVersionImportRepository
+{
+    public async Task SetCompleted(Guid dataSetVersionId,
```

Comment:
I'd already been thinking this for a while and I already prefer the way you've recommended here because if a change is made where the return type or method name is refactored but the first parameters don't change then no diff is generated in the history for that parameter making it easier to spot changes / blame etc.

I have changed my settings now, thanks.

I didn't realise I was the only one still adding the first param on the same line, and if I did I would have switched before now 🤣 .

Tbh I was hoping the code style changes we made recently were going to enforce this one way or the other but I don't think that's the case.